Makefile

```diff
@@ -54,6 +54,10 @@ endif
 
 GENERAL_TAGS := 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo libbpf '
 GPU_TAGS := ' gpu '
+ifeq ($(shell ldconfig -p | grep -q libhlml.so && echo exists),exists)
+	GPU_TAGS := $(GPU_TAGS)'habana '
```

Comment:
Do we expect a habana and an nvidia gpu to be installed in the same system? if not then we can probably use one flag or the other rather than bot wdyt?

Reply:
this is a good question! Let's have a follow up on the potential use cases. If we build all the GPU support in the same image, that'll be quite big. But maybe in certain use cases, image sizes don't matter.